test/framework/bootstrap/kind_provider.go

```diff
@@ -37,7 +37,7 @@ const (
 	DefaultNodeImageRepository = "kindest/node"
 
 	// DefaultNodeImageVersion is the default Kubernetes version to be used for creating a kind cluster.
-	DefaultNodeImageVersion = "v1.27.0"
+	DefaultNodeImageVersion = "v1.27.3"
```

Comment:
As a follow-up (which has to go on main first)

Should this include a sha as well?

Reply:
Yeah - good idea. I'll open a PR.